src/css/adoc/table.css

```diff
@@ -208,6 +208,20 @@ div.panels table {
     overflow: auto;
     width: 100%;
 
+    &.grid-none {
+      & > * > tr {
+        border: none !important;
+
+        & > td {
+          border: none !important;
```

Comment:
don't need `!important` here.  Ok to keep `!important` in line 213 because it overrides the line 377 (it's technically possible to not use important there, but it may get too complicated)

Reply:
Looks like she doesn't want the inner rows and columns so I am not sure why do we have to implement grid-rows and grid-cols as it is overriding grid-none. grid-none should be sufficient